manifests/plugin/ansible.pp

```diff
@@ -50,11 +53,13 @@
   String $callback = $foreman_proxy::plugin::ansible::params::callback,
   String $runner_package_name = $foreman_proxy::plugin::ansible::params::runner_package_name,
   Array[Stdlib::Absolutepath] $collections_paths = $foreman_proxy::plugin::ansible::params::collections_paths,
+  Enum['foreman', 'proxy'] $report_type = $foreman_proxy::plugin::ansible::params::report_type,
```

Comment:
Just thinking out loud: should `proxy` type automatically enable the host reporting feature? The only downside is that it wouldn't get disabled again if the users switches it to `foreman` again. I'd be OK with adding a line to the parameter documentation that tells the user to enable the host_reports plugin (both the Foreman and Proxy parts)

Reply:
> The whole feature (**for Ansible**) will work only if both ways are applied. For Puppet the first one only will do (but with additional configuration of puppet client/server).

Right, so I'm wondering if you set this value to `proxy`, should we do something about warning the user they'll have an incompatible setup somehow. But as I said: I'm fine with adding a line to the parameter documentation in this class that's something like "If set to proxy, the host_reports plugin must be enabled".

Reply:
Oh, right, will do.